osd/rosa_customer_pdb_preventing_drain.json

```
@@ -3,7 +3,7 @@
   "service_name": "SREManualAction",
   "log_type": "cluster-configuration",
   "summary": "Action required: Pod Disruption Budget preventing Node Drain",
-  "description": "Your cluster is attempting to drain node `${NODE}` but there is a pod disruption budget set that is preventing the drain. The OSD SRE team has identified the pod as `${POD}` running in the namespace: `${NAMESPACE}`. Please re-schedule this pod so that the node can drain. For more information on pod disruption budgets, see the documentation: https://access.redhat.com/documentation/en-us/red_hat_openshift_service_on_aws/4/html/nodes/working-with-pods#nodes-pods-pod-distruption-about_nodes-pods-configuring#.",
+  "description": "Your cluster is attempting to drain node `${NODE}` but there is a pod disruption budget set that is preventing the drain. The ROSA SRE team has identified the pod as `${POD}` running in the namespace: `${NAMESPACE}`. Please re-schedule this pod so that the node can drain. For more information on pod disruption budgets, see the documentation: https://access.redhat.com/documentation/en-us/red_hat_openshift_service_on_aws/4/html/nodes/working-with-pods#nodes-pods-pod-distruption-about_nodes-pods-configuring.",
```

```suggestion
  "description": "Your cluster is attempting to drain node `${NODE}` but there is a pod disruption budget set that is preventing the drain. The Red Hat SRE team has identified the pod as `${POD}` running in the namespace: `${NAMESPACE}`. Please re-schedule this pod so that the node can drain. For more information on pod disruption budgets, see the documentation: https://access.redhat.com/documentation/en-us/red_hat_openshift_service_on_aws/4/html/nodes/working-with-pods#nodes-pods-pod-distruption-about_nodes-pods-configuring.",
```

I would recommend removing Product name altogether and replacing it with `Red Hat`, which is aligned with other existing SL templates.